apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -653,6 +654,14 @@ export class InstallManager {
         }
       }
 
+      if (!isEmpty(packageJson.resolutions)) {
+        tempPackageJson.resolutions = tempPackageJson.resolutions || {};
```

Comment:
You should be able to remove this line as we already know `packageJson.resolutions` is non-empty and it's being assigned to `tempPackageJson.resolutions` on line 661.

Reply:
fixed